test-suite/buildtests/layer-04-noauth-everything.opts

```diff
@@ -42,11 +42,9 @@ MAKETEST="distcheck"
 #	We can't test them automatically everywhere without detecting those
 #	optional packages first.
 #
-#   --enable-ecap \
 #   --enable-epoll \
 #   --enable-kqueue \
 #   --enable-win32-service \
-#   --with-valgrind-debug \
 #   --with-ldap \
```

Comment:
FWIW, I could not find other optional features that satisfy all of these criteria:

* require dependencies that are likely to be missing in many build environments, preventing their unconditional enabling in layers (e.g., installed libecap)
* disabled by default (e.g., eCAP requires `--enable-ecap`)
* likely to build successfully when a simple pkg-config test passes (e.g., `pkg-config --exists valgrind`)

If there are such features not covered by this PR, they should be added.

This comment does not request any changes.